README-CN.md

```diff
@@ -41,11 +41,25 @@
 
 [Nebula Graph Cloud Service](https://cloud.nebula-graph.com.cn/) 是 Nebula Graph 的图数据库云服务平台（Database-as-a-Service，DBaaS），支持一键部署 Nebula Graph。Nebula Graph Cloud Service 服务目前处于公测阶段。公测期间不会就云服务收取任何费用，欢迎 **免费试用**。
 
+## 关于这个 Repo
+
+Please note that this repo is only for Nebula Graph `v1.*.*`.
+
+请注意当前 Repo(https://github.com/vesoft-inc/nebula) 是 Nebula Graph 的旧版本（ `v1.*.*`）仓库。
+
+从 `v2.*.*`开始（2021年四月发布了 GA），Nebula Graph 的代码被分到了三个仓库中： [nebula-graph](https://github.com/vesoft-inc/nebula-graph)，[nebula-common](https://github.com/vesoft-inc/nebula-common) 和 [nebula-storage](https://github.com/vesoft-inc/nebula-storage)。
+
+我们推荐您从访问 [nebula-graph](https://github.com/vesoft-inc/nebula-graph) 开始。
+
 ## 快速使用
 
-请查看[快速使用手册](https://docs.nebula-graph.com.cn/manual-CN/4.contributions/how-to-contribute/)，开始使用 **Nebula Graph**。
+请查看[快速使用手册](https://docs.nebula-graph.com.cn/2.0.1/2.quick-start/1.quick-start-workflow/)，开始使用 **Nebula Graph**。
```

Comment:
Since this is the repo of Nebula v1.x, should we just place the Quick Start of v1.0?

Reply:
For Quick Start Part, it's actually only applied to newcomers here, when the latest version makes more sense to them, and for those who (for some reason, which I consider can barely be the case) actually would like to access the `Quick Start` for the old version 1, it's also highlighted in the `*note*` part.

What do you think about this perspective?

Reply:
I think now the nebula repo somehow becomes the first impression entry that accidentally being accessed by first-comers, in a not small chance(it was an expected first impression with a decent amount of stars, right?).

It's more than it is on `README for version 1 nebula graph`. Thus it may be better to highlight contexts around a. version & hey, this is the old one, and b. ensure navigations are valid as the first impression of a project.

Reply:
There will be a lot of maintenance work if you put the link to the "latest" version here because the "latest" version changes over time. What about when the v3.x comes out? You will have to keep updating the link here, which is a tedious work to do. My point here is that we put the exact version of Quick Start here and keep visitors informed that this is the 1.x version of Nebula Graph and head to the nebula-graph repo if you are interested in a higher version. Let's keep things clear and simple. What do you think?

Reply:
@jamieliu1023 @wey-gu
I think the better idea is not to add the latest version in the URL.

The doc always pointer points to the latest version.

but users can select the other versions

Something like below.
https://pytorch.org/docs/stable/backends.html
https://zh-hans.reactjs.org/versions/

Reply:
> @jamieliu1023 @wey-gu
> I think the better idea is not to add the latest version in the URL.
> 
> The doc pointer always points to the latest version.
> 
> but users can select the other versions
> 
> Something like below.
> https://pytorch.org/docs/stable/backends.html
> https://zh-hans.reactjs.org/versions/

I agree with the latest aliases(also what's in my head these days, not cheating) ;-).
I will recommend/issue it to the docs, there is such a thing already on the tooling we are using.
https://squidfunk.github.io/mkdocs-material/setup/setting-up-versioning/#publishing-a-new-version

Reply:
> There will be a lot of maintenance work if you put the link to the "latest" version here because the "latest" version changes over time. What about when the v3.x comes out? You will have to keep updating the link here, which is a tedious work to do. My point here is that we put the exact version of Quick Start here and keep visitors informed that this is the 1.x version of Nebula Graph and head to the nebula-graph repo if you are interested in a higher version. Let's keep things clear and simple. What do you think?

I agree with your concerns, maybe let's see if we can have the "latest" alias in a short time soon on vesoft-inc/nebula-docs-cn#462